hoomd/write/dcd.py

```diff
@@ -0,0 +1,104 @@
+import hoomd
+from hoomd import _hoomd
+from hoomd.filter import ParticleFilter, All
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.operation import Writer
+
+
+class DCD(Writer):
+    R""" Writes simulation snapshots in the DCD format
```

Comment:
```suggestion
    """Writes simulation snapshots in the DCD format.
```
PEP8 docstring formatting conventions.